### PR TITLE
added github action build script with netlify push

### PR DESCRIPTION
**Pull request by chanceeakin**
Looks like y'all were manually pushing build artifacts to netlify. Spent the past few hours figuring this out, and thought I'd share it!

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+jobs:
+  Deploy:
+    name: "Deploy"
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install latest nightly
```
Comment (jstarry) [inline .github/workflows/rust.yml]: nit: please use stable

File: .github/workflows/rust.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+name: Build and Deploy Site
+
+env:
+  NODE_VERSION: "12.x"
```
Comment (jstarry) [inline .github/workflows/rust.yml]: Let's use 14

**Review by jstarry — COMMENTED** [review]
This is awesome, thanks for adding this @chanceeakin! Do you mind addressing the two nits?

File: .github/workflows/rust.yml (hunk 1 above)
Comment (chanceeakin) [reply .github/workflows/rust.yml]: Changed to stable. Didn't realize rust had adopted wasm bindings/compile targets in stable already. Good call out there! :)

File: .github/workflows/rust.yml (hunk 2 above)
Comment (chanceeakin) [reply .github/workflows/rust.yml]: Changed to latest LTS. 

dangerous

**Comment by chanceeakin** [comment]
@jstarry: You're welcome! Nits are addressed and gifs added where appropriate. 😂

**Comment by chanceeakin** [comment]
AFAIK, this is good to go, but I'm unsure of why the travis CI build keeps failing.

**Pull request merged**